lib/danger/request_sources/github.rb

```diff
@@ -73,26 +77,31 @@ def submit_pull_request_status!(warnings: nil, errors: nil, details_url: nil)
       if errors.count > 0
         # We need to fail the actual build here
         abort("danger found #{errors.count} error(s) and doesn't have write access to the PR")
+      else
+        puts message
```

Comment:
I wasn't sure if it had finished when a build was completed, figured others would have the same issue